src/loaders/meson.build

```diff
@@ -1,24 +1,28 @@
 subloader_dep = []
 
+if get_option('loaders').contains('tvg') == true
+    subdir('tvg')
+endif
+
 if get_option('loaders').contains('svg') == true
     subdir('svg')
 endif
 
-if get_option('loaders').contains('png') == true
+subdir('external_png')
+
+if not png_dep.found()
     subdir('png')
 endif
 
-if get_option('loaders').contains('tvg') == true
-    subdir('tvg')
-endif
+subdir('external_jpg')
 
-if get_option('loaders').contains('jpg') == true
+if not jpg_dep.found()
     subdir('jpg')
```

Comment:
I think the external_jpg folder and the jpg folder should have an `or` relationship.

```
if not jpg_dep.found()
  subdir('jpg')
else
  subdir('external_jpg')
endif
```

Reply:
Probably, but we don't have dynamic linking mechanism yet.
Right now, we do support either module. it's enough to so far.